.dependency-lock/pom.xml

```diff
@@ -473,6 +482,25 @@
       <optional>false</optional>
       <dependency-lock:integrity>sha512:aMXfAa1SKRfZVLYss7ETdrNEaSDo1cs1c33lCOas2Ro2s0FEkz9Ce74dKdypCXup3SQxxxP+GAafJxqAtRnwUQ==</dependency-lock:integrity>
     </dependency>
+    <dependency>
+      <groupId>org.exist-db.thirdparty.org.eclipse.wst.xml</groupId>
+      <artifactId>xpath2</artifactId>
+      <version>1.2.0</version>
+      <type>jar</type>
+      <scope>test</scope>
+      <optional>false</optional>
+      <dependency-lock:integrity>sha512:yaoKSo/Fa/Kms3kNsHGzYqkpMNav1CWd/nOiLAeMwZ88GhiCmA78IOUo1I5HeWS1izWPesxxTmCmvZOnT5SZOw==</dependency-lock:integrity>
+    </dependency>
+    <dependency>
+      <groupId>org.exist-db.thirdparty.xerces</groupId>
+      <artifactId>xercesImpl</artifactId>
```

Comment:
Manually crafted XML Schema 1.1 Supporting Xerces Implementation wrapper (https://blog.adamretter.org.uk/xerces-xsd11-maven/ for the credits)